routers/repo/http.go

```diff
@@ -276,6 +277,19 @@ func HTTP(ctx *context.Context) {
 		}
 	}
 
+	if !repoExist {
+		repo, err = models.CreateRepository(authUser, authUser, models.CreateRepoOptions{
```

Comment:
```suggestion
		if owner, err := GetUserByName(string(b)); err != nil {
			return ctx.HandleText(http..... some appropriate error)
		}
		repo, err = models.CreateRepository(authUser, owner, models.CreateRepoOptions{
```

Reply:
I had noticed that I was not using `owner` as well.  
I have some updates coming that should alleviate the issue.

Reply:
I meant to use `username` instead of `b` (bad copy & paste) to support orgs, as @davidsvantesson suggested.

Reply:
No problem. `owner` is actually already loaded [earlier in this func](https://github.com/go-gitea/gitea/blob/master/routers/repo/http.go#L95)